### PR TITLE
Remove letstune.Metric class

### LINKED ISSUE #7: Make metric always greater-is-better
Currently `Metric` class has a field `greater_is_better`. Force the field to be always `true`.

**Why?**

This would significantly simplify codebase.

**User impact**

Usually metrics in hyper-parameter tuning are `greater_is_better` (accuracy, F1 score, etc.).
User can easily create negated value of a loss if needed (like `neg_mean_square_error`).

Scikit-learn has similar convention in [sklearn.metrics.make_scorer](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.make_scorer.html).

### PR DESCRIPTION
**Pull request by mslapek**
Fixes #7 and #8

### CODE REVIEW

**Comment by mslapek** [comment]
LGTM
